multi_step_wizard/models/multi_step_wizard.py

```diff
@@ -51,6 +55,15 @@ def open_next(self):
         state_method()
         return self._reopen_self()
 
+    def open_previous(self):
```

Comment:
This will break all dependant modules right now as they don't define this method. The back button should be hidden unless method is available

Reply:
Yes, I completely agree to this. Added a computed field that shows the back button only if its implemented.

Reply:
We should make similar change in v11 as well once v12 is wrapped up